packages/devextreme-angular/README.md

```diff
@@ -1,26 +1,24 @@
-[![CircleCI](https://img.shields.io/circleci/project/github/DevExpress/devextreme-angular/master.svg)](https://circleci.com/gh/DevExpress/devextreme-angular) [![npm version](https://badge.fury.io/js/devextreme-angular.svg)](https://badge.fury.io/js/devextreme-angular)
+# DevExtreme (Angular add-on)
 
+[![NPM version](https://img.shields.io/npm/v/devextreme-angular.svg?maxAge=43200)](https://www.npmjs.com/package/devextreme-angular)
 
-# DevExtreme Angular UI Components #
+The `devextreme-angular` package allows you to use [DevExtreme components](http://js.devexpress.com/Demos/WidgetsGallery/) in [Angular](https://angular.io/) applications.
```

Comment:
Wrong message. We do not allow to use DevExtreme in Angular apps. We offer a separate component suite for Angular (and other frameworks).